### ISSUE TITLE
No diagnostic for commas in initializer

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1655]
I expected a diagnostic for the following code, but none was reported:

```csharp
List<int> x = new List<int>
{
    0
    , 1
};
```

Should one of the existing diagnostics apply to this case?

**Comment by vweijsters** [comment]
The old StyleCop flags this with SA1001, which seems logical to me.

**Comment by sharwell** [comment]
Is the documentation for SA1001 incorrect? In particular, should a comma ever be allowed at the beginning of a line?

**Comment by Przemyslaw-W** [comment]
I would also expect some kind of SA1113, but targeted at initializers

**Comment by vweijsters** [comment]
I just looked at the old StyleCop code and that code will definitely not accept a comma at the start of the line, so I say that the documentation is wrong.

**Comment by sharwell** [comment]
I filed :bug: #1663 to correct.
